compiler/luci/export/src/CircleTensorExporter.cpp

```diff
@@ -361,6 +361,42 @@ flatbuffers::Offset<circle::Buffer> encodeOpBufferByDType(FlatBufferBuilder &bui
   return CreateBuffer(builder, array_offset);
 }
 
+template <>
+flatbuffers::Offset<circle::Buffer>
+encodeOpBufferByDType<loco::DataType::S4>(FlatBufferBuilder &builder, luci::CircleConst *c)
+{
+  const uint32_t size = c->size<loco::DataType::S4>();
+  const uint32_t raw_size = (size + 1) / 2;
```

Comment:
I renamed variables for clarification
 
```suggestion
  const uint32_t num_elems = c->size<loco::DataType::S4>();
  const uint32_t buffer_size = (size + 1) / 2;
```

Reply:
used `size` as existing `encodeOpBufferByDType` used `size`
but for `raw_size`, I'll use `buffer_size`

Reply:
> but for raw_size, I'll use buffer_size

existing codes also used `raw_data` so it looks using `raw_size` would be better.